### PR TITLE
Create dynamodb endpoint for vpc

### PR DESCRIPTION
**Pull request by mindmorass**
- same logic as s3 endpoint but for dynamodb
- outputs for endpoint ids

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -127,7 +127,7 @@ data "aws_vpc_endpoint_service" "s3" {
   service = "s3"
 }
 
-resource "aws_vpc_endpoint" "ep" {
+resource "aws_vpc_endpoint" "s3-ep" {
```
Comment (antonbabenko) [inline main.tf]: Let's rename it to `s3` and the one you are adding will be `dynamodb`.

File: outputs.tf [hunk 2]
```diff
@@ -61,3 +61,11 @@ output "igw_id" {
 output "default_network_acl_id" {
   value = "${aws_vpc.mod.default_network_acl_id}"
 }
+
+output "vpc_s3_endpoint_id" {
+  value = "aws_vpc_endpoint.s3-ep.id"
```
Comment (antonbabenko) [inline outputs.tf]: Value should be like `${aws_vpc_endpoint.s3.id}`.

File: outputs.tf [hunk 3]
```diff
@@ -61,3 +61,11 @@ output "igw_id" {
...
+}
+
+output "vpc_dynamodb_endpoint_id" {
+  value = "aws_vpc_endpoint.dynamodb-ep.id"
```
Comment (antonbabenko) [inline outputs.tf]: Value should be like `${aws_vpc_endpoint.dynamodb.id}`.

File: main.tf (hunk 1 above)
Comment (antonbabenko) [reply main.tf]: Also, we normally do not use dashes inside names.

File: outputs.tf [hunk 4]
```diff
@@ -61,3 +61,11 @@ output "igw_id" {
 output "default_network_acl_id" {
   value = "${aws_vpc.mod.default_network_acl_id}"
 }
+
+output "vpc_s3_endpoint_id" {
```
Comment (antonbabenko) [inline outputs.tf]: Change name to be similar to resource - `vpc_endpoint_s3_id`.

File: outputs.tf [hunk 5]
```diff
@@ -61,3 +61,11 @@ output "igw_id" {
...
+  value = "aws_vpc_endpoint.s3-ep.id"
+}
+
+output "vpc_dynamodb_endpoint_id" {
```
Comment (antonbabenko) [inline outputs.tf]: Please add missing outputs to README. There are some of the outputs missing in that file.

File: main.tf (hunk 1 above)
Comment (mindmorass) [reply main.tf]: update

File: outputs.tf (hunk 2 above)
Comment (mindmorass) [reply outputs.tf]: fixed

File: outputs.tf (hunk 5 above)
Comment (mindmorass) [reply outputs.tf]: added outputs to documentation

File: outputs.tf (hunk 3 above)
Comment (mindmorass) [reply outputs.tf]: updated outputs

File: outputs.tf (hunk 4 above)
Comment (mindmorass) [reply outputs.tf]: updated names to match resource order

File: outputs.tf [hunk 6]
```diff
@@ -61,3 +61,11 @@ output "igw_id" {
 output "default_network_acl_id" {
   value = "${aws_vpc.mod.default_network_acl_id}"
 }
+
+output "vpc_endpoint_s3_id" {
+  value = "aws_vpc_endpoint.s3.id"
```
Comment (antonbabenko) [inline outputs.tf]: The value should be wrapped with `${...}`. See line 62, for example.

**Comment by antonbabenko** [comment]
New release is [v1.0.13](https://github.com/terraform-community-modules/tf_aws_vpc/releases/tag/v1.0.13)

**Comment by antonbabenko** [comment]
Thank you for the contribution!

**Pull request merged**